plugins/modules/ntnx_pc_unregistration_v2.py

```diff
@@ -0,0 +1,248 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# Copyright: (c) 2025, Nutanix
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+from __future__ import absolute_import, division, print_function
+
+__metaclass__ = type
+
+DOCUMENTATION = r"""
+module: ntnx_pc_unregistration_v2
+short_description: Unregister a PC-PC setup connected using availability zone.
+version_added: 2.1.0
+description:
+    - Unregister a PC-PC setup connected using availability zone.
+    - This module cannot be used to unregister PC-PE.
+    - This module uses PC v4 APIs based SDKs
+options:
+    wait:
+        description:
+            - Wait for the task to complete.
+        type: bool
+        required: False
+    pc_ext_id:
+        description:
+            - External ID of the local cluster.
```

Comment:
Please explain properly what pc_ext_id is and which module can be used to fetched. I am not sure if its a external ID of local cluster. That gives wrong message.

Same for `ext_id`